docs/modules/ROOT/pages/troubleshooting/index.adoc

```diff
@@ -157,3 +157,47 @@ kubectl get secrets -o json |
         {auths: .}
     '
 ----
+
+== Manually Update Task Bundles
+
+Usually, Konflux users rely on link:https://docs.renovatebot.com/[renovate] to update
```

Comment:
@staticf0x , I think that you have some action item to move MintMaker documentation into this repo. We should update this comment/link during that process.